Assets/Game/Scripts/Explodables/ExplodableBase.cs

```diff
@@ -14,48 +13,62 @@ public abstract class ExplodableBase : MonoBehaviour, IExplodable {
         [SerializeField]
         bool _isObjective;
 
+        /* Removing this for now. The base poppable can be extended to add colliders or use physics calls.
+         The base poppable will use a sphere cast
         [Tooltip("You can use any explosion shape you want. The trigger area just needs to have a TriggerExplodableTracker on it.")]
         [SerializeField]
-        TriggerExplodableTracker _explosionTrigger;
+        TriggerExplodableTracker _explosionTrigger;*/
 
         public bool AutoExplode => _autoExplode;
         public bool IsObjective => _isObjective;
-        public bool IsPrimed { get; private set; }
+        public bool IsPrimed { get; private set; } // now only used for scoring
 
-        void Start () {
+        protected virtual void Start () {
             _collection = GameController.Instance.Explodables;
             _collection.Add(this);
         }
 
         public void Explode () {
-            StartCoroutine(ExplosionLoop());
+            //StartCoroutine(ExplosionLoop());
+            PopManager.Instance.AddPopToQueue(GetComponent<Poppable>()); // always extended by poppable. Could rework, no time
         }
 
-        IEnumerator ExplosionLoop () {
+        public void Prime () {
+            IsPrimed = true;
+            // Track this globally so we can wait for all explosions to resolve
+            _collection.AddExploding(this);
+            PlayAnimation(); // may hide this for now?
+
+        }
+
+        /*IEnumerator ExplosionLoop () {
```

Comment:
Can we delete all the commented code out? It makes the file hard to read and review.